styles/typography.less

```diff
@@ -18,3 +18,40 @@ a {
     color: @primary;
   }
 }
+
+/* Font-size for pages' h1 header; overwriting Bootstrap */
+
+div.page-header h1 {
+  font-size: @page-header-size;
```

Comment:
We use tabs for indentation instead of spaces.

Reply:
Hmm, I'm using PHPStorm with automatic tab indentation... Will check.

Reply:
Ahh, sorry. That was not configured for LESS.

Reply:
:+1: It looks like there are still a couple of places with spaces in `typography.less`.

Reply:
Thanks. I see. Suppose auto-indent didn't work for some reason. Will do it manually.

Reply:
Done.